src/EnergyPlus/OutAirNodeManager.cc

```diff
@@ -363,7 +363,6 @@ namespace OutAirNodeManager {
 
                 if (NumAlphas > 1) {
                     AnyLocalEnvironmentsInModel = true;
-                    Node(NodeNums(1)).IsLocalNode = true;
```

Comment:
Delete both of these lines here (IsLocalNode already deleted), but delete AnyLocalEnvironments line.

Reply:
@mjwitte This is the original code. Let me upload the fix in EMSManager first. If everything is OK, I will delete this line.

Reply:
@Myoldmopar Remember I'd like this line deleted also.

Reply:
It looked like it was...sorry...I'll delete that line locally and run the test suite.  If it passes I'll push up my updated branch and then merge.